src/frontend/ExtensionManager/index.tsx

```diff
@@ -26,27 +27,48 @@ const ExtensionManager = function () {
   const rootRef = useRef<HTMLDivElement>(null)
   const trueAsStr = 'false' as unknown as boolean | undefined
 
+  const mmContainerStyle = {} as React.CSSProperties
+
+  if (OverlayState.title === 'HyperPlay Extension') {
+    mmContainerStyle.left = 0
+    mmContainerStyle.right = 'unset'
+    mmContainerStyle.top = 0
+    // document.documentElement.style.setProperty('--top-navbar-height', `0px`)
```

Comment:
We should remove commented code, we can always access that through git version control if we need later 🙌

Reply:
sure. accidentally left it in from debugging